cdk/requirements.txt

```diff
@@ -78,8 +78,9 @@ pycparser==2.20
 python-dateutil==2.8.2
 PyYAML==5.3.1
 rsa==4.5
-s3transfer==0.3.7
+s3transfer==0.5.1
 six==1.16.0
 toml==0.10.2
 typing-extensions==3.10.0.2
 urllib3==1.26.7
+zipp==3.7.0
```

Comment:
I see in [pypi](https://pypi.org/project/zipp/) that zipp is:

> A pathlib-compatible Zipfile object wrapper. Official backport of the standard library [Path object](https://docs.python.org/3.8/library/zipfile.html#path-objects).

What are you using this for?

Reply:
Good question. Let me see if I can track down what caused it to be installed.